projects/assets/src/translations/en/my-account.ts

```diff
@@ -36,14 +36,14 @@ export const myAccount = {
       returnAction: 'Request a Return',
       cancelAction: 'Cancel Order Items',
       item: 'Item',
-      itemPrice: 'Price',
-      quantity: 'Qty',
-      returnQty: 'Return Qty',
-      cancelQty: 'Cancel Qty',
-      returnAll: 'Return All',
-      cancelAll: 'Cancel All',
+      itemPrice: 'Item Price',
+      quantity: 'Max Quantity',
+      returnQty: 'Quantity to Return',
+      cancelQty: 'Quantity to Cancel',
+      setAll: 'Set all quantities to maximum',
```

Comment:
Wrong line for the other comment, the mocks show this text "Return complete order"

Reply:
https://wiki.hybris.com/pages/viewpage.action?spaceKey=spar&title=Cancellations+and+Returns+UX+MVP+1+for+Dec2019. And for this one, I confirmed with PO.